packages/mui-base/src/TextareaAutosize/TextareaAutoSize.test.tsx

```diff
@@ -62,12 +63,12 @@ describe('<TextareaAutosize />', () => {
 
       it('should handle the resize event', () => {
         const { container } = render(<TextareaAutosize />);
-        const input = container.querySelector('textarea[aria-hidden=null]');
+        const input: HTMLInputElement | null = container.querySelector('textarea[aria-hidden=null]');
```

Comment:
the input is supposed to exist, there is something else wrong if it doesn't.

```suggestion
        const input: HTMLInputElement = container.querySelector('textarea[aria-hidden=null]');
```